packages/@webex/plugin-cc/test/unit/spec/AgentConfig/AgentConfig.ts

```diff
@@ -150,11 +152,11 @@ describe('AgentConfig', () => {
 
     const result = await agentConfig.getAgentProfile();
 
-    expect(getUserUsingCISpy).toHaveBeenCalledOnceWith;
-    expect(getDesktopProfileByIdSpy).toHaveBeenCalledOnceWith;
-    expect(getListOfTeamsSpy).toHaveBeenCalledOnceWith;
-    expect(getListOfAuxCodesSpy).toHaveBeenCalledOnceWith;
-
+    expect(getUserUsingCISpy).toHaveBeenCalledTimes(1);
```

Comment:
Please add the argument check using to haveBeenCalledWith

Reply:
Added!